pkg/container/docker.go

```diff
@@ -83,7 +83,9 @@ func (p *DockerLoginRegistry) Execute(runtime connector.Runtime) error {
 	auths := registry.DockerRegistryAuthEntries(p.KubeConf.Cluster.Registry.Auths)
 
 	for repo, entry := range auths {
-
+		if len(entry.Username) == 0 || len(entry.Password) == 0 {
```

Comment:
If all elements of the array are `continue`, then the `docker login` command will not be executed and the `$HOME/.docker/config.json` file will not be generated. And the following comand `mkdir -p /.docker && cp -f $HOME/.docker/config.json /.docker/ && chmod 0644 /.docker/config.json` will report a "Not Found" error.

Reply:
add judgement for $HOME/.docker/config.json